fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/rdf/impl/RootRdfContext.java

```diff
@@ -129,7 +132,25 @@ private void concatRepositoryTriples() {
                     createTypedLiteral(counters.get(PREFIX + FIXITY_REPAIRED_COUNTER).getCount()).asNode()));
         }
 
+        //add project build description triples here
+        final BuildPropertiesLoader buildLoader = new BuildPropertiesLoader();
+
+        final com.hp.hpl.jena.graph.Node releaseSubject = createURI(REPOSITORY_NAMESPACE + "release");
+            b.add(create(subject(), DOAP_RELEASE.asNode(), releaseSubject));
+            b.add(create(releaseSubject, type.asNode(), createURI(DOAP_NAMESPACE + "Version")));
```

Comment:
To be clear, currently, every time the command `mvn install` is run three pieces of information are dynamically created and added to the HTML splash page:
* `project.version` e.g. 4.4.1-SNAPSHOT
* `build.timestamp` e.g. 2015-11-29
* `buildNumber` e.g. 80571244

This ticket is to provide that same information in a machine-readable fashion... hence the initial suggestion of triples on the repository root resource.